dandi/tests/test_validate.py

```diff
@@ -24,7 +24,7 @@ def test_validate_relative_path(
     monkeypatch.chdir(selected_dataset_path)
     # improper relative path handling would fail with:
     # ValueError: Path '.' is not inside Dandiset path '/tmp/.../asl003'
-    validate(".")
+    list(validate("."))
```

Comment:
oh, nice catch.

@TheChymera did you actually check that this test failed on master without the fix?  it is a nice "habbit" to have/test that way.  You can `git checkout master` even without committing your test thus to check if your test catching original problem, and then `git checkout -`, make sure that now it passes, and then `git commit`